.github/workflows/continuous-integration.yml

```diff
@@ -26,3 +26,26 @@ jobs:
         run: sudo xargs apt-get install -y < .github/workflows/apt-packages.txt
 
       - run: ci-tests/test-spike
+
+      - name: Calver Release
+        uses: StephaneBour/actions-calver@master
```

Comment:
Who is StephaneBour?

Reply:
The github action file is provided by that user : https://github.com/StephaneBour/actions-calver